src/MOI/MOI_wrapper.jl

```diff
@@ -2322,6 +2376,7 @@ function MOI.optimize!(model::Optimizer)
     # cache rhs: must be done before hand because it cant be
     # properly queried if the problem ends up in a presolve state
     rhs = Xpress.getrhs(model.inner)
+    is_mip(model) && _update_MIP_start!(model)
```

Comment:
```julia
if is_mip(model)
    _update_MIP_start!(model)
end
```

Reply:
Also, it might be a little expensive trying to build the solution for every MIP. I will have to think more about it. May I should add an option later on.

Reply:
Yes, I agree, probably `set_start_value` should turn a flag `has_a_mip_start` in the `Model` to `true` and the model should only try to get the MIP-start if this is the case. The flag could be turned off after the solution was already added. However, I find the design of JuMP/MOI MIP-starts a little strange. There is some way of specifying multiple MIP-starts or not?

Reply:
Currently, only one start is accepted.
It would be interesting to propose the change to multiple starts in the MOI repo.

Reply:
For now I do not have the time for this, but maybe in the future is an interesting problem to tackle.

Is the expensive solution building blocking this PR? Should I do something about it?